src/code_generation.jl

```diff
@@ -10,7 +10,7 @@ end
 
 # GeneratedFunction(code::Expr) = GeneratedFunction(eval(code), code)
 
-(f::GeneratedFunction{F})(x...) where {F} = f.f(x...)
+#(f::GeneratedFunction{F})(x...) where {F} = f.f(x...)
```

Comment:
Why did you comment out this line?

Reply:
If you uncomment this again then you can call `Contractor`

Reply:
Yes thats by mistake.